dataland-frontend/src/components/forms/parts/UploadReports.vue

```diff
@@ -138,14 +186,18 @@ export default defineComponent({
      * Initializes the already uploaded reports from provided reports
      */
     prefillAlreadyUploadedReports() {
-      if (this.referencedReportsForPrefill) {
-        for (const key in this.referencedReportsForPrefill) {
+      const sourceOfReferencedReportsForPrefill = (this.referencedReportsForPrefill ??
+        this.injectReferencedReportsForPrefill) as ObjectType;
+
+      if (sourceOfReferencedReportsForPrefill) {
+        for (const key in sourceOfReferencedReportsForPrefill) {
+          const referencedReport = (sourceOfReferencedReportsForPrefill as { [key: string]: CompanyReport })[key];
           this.storedReports.push({
             reportName: key,
-            reference: this.referencedReportsForPrefill[key].reference,
-            currency: this.referencedReportsForPrefill[key].currency,
-            reportDate: this.referencedReportsForPrefill[key].reportDate,
-            isGroupLevel: this.referencedReportsForPrefill[key].isGroupLevel,
+            reference: referencedReport.reference,
+            currency: referencedReport.currency,
+            reportDate: referencedReport.reportDate,
+            isGroupLevel: referencedReport.isGroupLevel,
           });
         }
```

Comment:
Rename "duplicatesAmongReferenceableReportNames" to "areDuplicatesAmongReferenceableReportNames"

Reply:
Done.

Reply:
👍